futaba/sql/models/settings.py

```diff
@@ -17,12 +17,15 @@
 # False positive when using SQLAlchemy decorators
 # pylint: disable=no-value-for-parameter
 
+import enum
 import functools
 import logging
 
 import discord
-from sqlalchemy import BigInteger, Boolean, Column, Table, Unicode
-from sqlalchemy import CheckConstraint, ForeignKey, SmallInteger
+
+from futaba.utils import partition_on
+from sqlalchemy import (BigInteger, Boolean, CheckConstraint, Column, Enum,
+                        ForeignKey, SmallInteger, Table, Unicode)
```

Comment:
`futaba.*` imports should be treated like they are relative, and go after third-party imports.
e.g.
```py
import discord
import toml

from futaba.utils import escape_backticks
from .local import helper_func
```